dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/CheckUrlHelperTest.java

```diff
@@ -17,54 +17,26 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import com.google.common.io.Files;
 import io.dockstore.common.MuteForSuccessfulTests;
-import io.dockstore.webservice.helpers.CheckUrlHelper.TestFileType;
-import io.dropwizard.testing.ResourceHelpers;
-import java.io.File;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
+import io.dockstore.webservice.helpers.CheckUrlInterface.UrlStatus;
 import java.util.Set;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import uk.org.webcompere.systemstubs.jupiter.SystemStub;
 import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
-import uk.org.webcompere.systemstubs.stream.SystemErr;
-import uk.org.webcompere.systemstubs.stream.SystemOut;
 
 @ExtendWith(SystemStubsExtension.class)
 @ExtendWith(MuteForSuccessfulTests.class)
 class CheckUrlHelperTest {
 
-    @SystemStub
-    public final SystemOut systemOut = new SystemOut();
-
-    @SystemStub
-    public final SystemErr systemErr = new SystemErr();
-
-    @Test
-    void getUrlsFromJSON() throws IOException {
-        File file = new File(ResourceHelpers.resourceFilePath("testParameterFile1.json"));
-        String s = Files.asCharSource(file, StandardCharsets.UTF_8).read();
-        Set<String> urls = CheckUrlHelper.getUrlsFromJSON(s);
-        assertEquals(3, urls.size(), "Should have two from GitHub, 1 from FTP");
-    }
-
-    @Test
-    void getUrlsFromYAML() throws IOException {
-        File file = new File(ResourceHelpers.resourceFilePath("testParameterFile1.yaml"));
-        String s = Files.asCharSource(file, StandardCharsets.UTF_8).read();
-        Set<String> urls = CheckUrlHelper.getUrlsFromYAML(s);
-        assertEquals(3, urls.size(), "Should have same amount of URLs as the JSON equivalent above");
-    }
-
     @Test
-    void getUrlsFromInvalidFile() throws IOException {
-        File file = new File(ResourceHelpers.resourceFilePath("valid_description_example.wdl"));
-        String s = Files.asCharSource(file, StandardCharsets.UTF_8).read();
-        assertTrue(CheckUrlHelper.checkTestParameterFile(s, "fakeBaseUrl", TestFileType.YAML).isEmpty());
-        assertTrue(CheckUrlHelper.checkTestParameterFile(s, "fakeBaseUrl", TestFileType.JSON).isEmpty());
+    void testCheckUrls() {
+        final CheckUrlHelper checkUrlHelper =
+            new CheckUrlHelper("https://this.url.is.unused.in.these.tests");
+        assertEquals(UrlStatus.ALL_OPEN, checkUrlHelper.checkUrls(Set.of()),
+            "No urls should return all open");
+        assertEquals(UrlStatus.NOT_ALL_OPEN, checkUrlHelper.checkUrls(Set.of("localfile.cram")), "A local file is a malformed url");
+        assertEquals(UrlStatus.NOT_ALL_OPEN, checkUrlHelper.checkUrls(Set.of("s3://human-pangenomics/working/HPRC_PLUS/HG005/raw_data/Illumina/child/5A1-24481579/5A1_S5_L001_R1_001.fastq.gz")),
```

Comment:
Phase 2 suggestion, we do have vfs2 which should recognize s3. 
(actually no, requires library, phase 2 anyway)

Reply:
#5397